### PR TITLE
Add Cheat Sheet

### PR DESCRIPTION
**Pull request by cknuepfer**
Solves #20

### CODE REVIEW

File: _extras/cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| `git add <file>`                           | adds `<file>` to staging area                                                                                                               |
+| `git add .`                                | adds all files to the staging area that where changed since last commit                                                                    |
+| `git commit -m "<commit-message>"`         | creates a commit from the current (staged) project state with the message `<commit-message>`                                               |
+| `git commit --amend -m "<commit-message>"` | replaces last commit using new commit message; additional files can be staged beforehand (`git add`)                                 |
+| `git gui`                                  | starts the graphical client for staging and committing                                                                                     |
+| file `.gitignore`                          | contains files ignored by Git<br>each line can contain a file name or patterns like `*.log` which will ignore all files ending with `.log` |
```
Comment (rgaiacs) [inline _extras/cheatsheet.md]: We should use `cat` instead of file.

File: _extras/cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+|----------------------------------------------------|-------------------|
+| `git config --global user.name "<username>"`       | set user name     |
+| `git config --global user.email "<email-address>"` | set email address |                                                   |                   |
+
+## First steps
+
+| command                                    | description                                                                                                                                |
+|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `git init <repo-name>`                     | creates a new repository called `<repo-name>`                                                                                               |
+| `git clone <<repository>`                  | clones an existing repository                                                                                                               |
```
Comment (rgaiacs) [inline _extras/cheatsheet.md]: Extra `<`.

File: _extras/cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+|----------------------|--------------------------------------------------------------------------------------------------|
+| `git reset <commit>` | set repository back into the state of `<commit>`, caution: deletes all commits after `<commit>`! |
+
+## Working with remote repositories
+
+| command                              | description                                             |
+|--------------------------------------|---------------------------------------------------------|
+| `git remote -v`                      | lists remotes                                           |
+| `git remote add <name> <url>`        | adds remote `<name>` pointing at `<url>`                |
+| `git remote set-url <name> <newurl>` | changes URL of remote `<name>` to `<newurl>`            |
```
Comment (rgaiacs) [inline _extras/cheatsheet.md]: Replace `newurl` with `new-url` to match the style used in the other commands.

**Review by rgaiacs — CHANGES_REQUESTED** [review]
Minor changes. Looks good to me.

**Comment by ivastar** [comment]
Thank you for your contribution @cknuepfer!

**Pull request merged**